pkg/ui/workspaces/cluster-ui/src/queryFilter/filter.tsx

```diff
@@ -564,7 +564,7 @@ export class Filter extends React.Component<QueryFilter, FilterState> {
         }))
       : [];
     const schemaInsightTypeValue = schemaInsightTypeOptions.filter(option => {
-      return filters.schemaInsightType.split(",").includes(option.label);
+      return filters.schemaInsightType?.split(",").includes(option.label);
```

Comment:
If we're changing this code anyways, can we create the array (or set) outside of the filter fn:
```
const selectedSchemaInsightTypes = filters.schemaInsightType?.split(",") ?? [];
const schemaInsightTypeValue = schemaInsightTypeOptions.filter(option => selectedSchemaInsightTypes.includes(option.label);
);
```

Reply:
Done! I went ahead and fixed all the other issues of the same type in the file since we see these errors in the console logs quite frequently.